content/docs/developer/_index.md

```diff
@@ -12,7 +12,7 @@ The steps in this section describe how to set up a local testnet for development
 
 It is assumed that you have:
 
-- [Setup the `aut` CLI](/account-holders/setup-autcli/) tool and have an [account](/account-holders/create-acct/).
+- [Setup the `autcli`](/account-holders/setup-autcli/) tool and have an [account](/account-holders/create-acct/).
```

Comment:
Should this be more consistent with the revised page title or linkTitle? E.g.

> \[Setup the Autonity CLI tool](...)

or

> \[Setup Autonity CLI](...)

or perhaps just include the word "tool" in the hyperlink?

Reply:
Correct. Edited to the revised page title. Commit: 1a94c1cac2b5aeb048dff0255286261c390bb78d